uspace/lib/c/arch/ia64/include/libarch/fibril.h

```diff
@@ -42,7 +42,7 @@
 #include <libarch/fibril_context.h>
 
 /*
- * context_save() and context_restore() are both leaf procedures.
+ * setjmp() and __longjmp() are both leaf procedures.
```

Comment:
The standard says setjmp is a macro.

Reply:
Ah, indeed, I forgot about that. `#define setjmp(env) setjmp(env)` should do the trick.